src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -27,7 +27,7 @@
 
 namespace OpenTelemetry.Exporter.Jaeger
 {
-    public class JaegerExporter : BaseExporter<Activity>
+    public class JaegerExporter : BaseExporter<Activity>, IProviderContainer<TracerProvider>
```

Comment:
can we do this in the baseexporter itself? So that exporters just access this.Provider.Resource, and baseexporters take care of populating Provider?

Reply:
Let me do an "Option 3 - Alternative" PR for that so we can see what it looks like?